apps/api/src/server/app/case-officer/case-officer.controller.js

```diff
@@ -25,7 +27,10 @@ const getAppealDetails = async function (request, response) {
 	return response.send(formattedAppeal);
 };
 
-const confirmingLPAQuestionnaire = function (request, response) {
+const confirmingLPAQuestionnaire = async function (request, response) {
+	const reviewResult = await reviewIncomplete(request.body);
```

Comment:
`reviewIncomplete` is not an async function, therefore we do not need an await here

Reply:
corrected